### PR TITLE
adds note about passing `format`

### PR DESCRIPTION
**Pull request by stuartromanek**

### CODE REVIEW

File: docs/reference/field-types/color.md [hunk 1]
```diff
@@ -38,6 +38,12 @@ themeColor: {
 <!-- |contextual | Boolean | false | If `true`, it will prevent the field from appearing in the editor modal | -->
 <!-- |readOnly | Boolean | false | If `true`, prevents the user from editing the field value | -->
 
+### Options
+General options you can pass to a `color` field.
+| Option | Type | Default | Description |
+|---------------|------|---------|-------------|
+| `format` | String | `hex8` | The color string format to be saved. Possible options are "rgb", "prgb", "hex6", "hex3", "hex8", "name", "hsl", "hsv". |
```
Comment (abea) [inline docs/reference/field-types/color.md:45]: this should be part of the "Color picker options" table below, right?

**Review by abea — CHANGES_REQUESTED** [review]

File: docs/reference/field-types/color.md (hunk 1 above)
Comment (abea) [reply docs/reference/field-types/color.md:45]: I'm not opposed to changing how it works in core. As it is currently built those picker options are documented below, including a code example showing how to nest it.

Comment (abea) [reply docs/reference/field-types/color.md:45]: It looks like this is related to the core PR https://github.com/apostrophecms/apostrophe/pull/3122/. I'll respond with more there.

Comment (abea) [reply docs/reference/field-types/color.md:45]: Actually, see the area field page for how we're already doing this. The format is different, giving more space to the option.

**Review by abea — CHANGES_REQUESTED** [review]

File: docs/reference/field-types/color.md (hunk 1 above)
Comment (stuartromanek) [reply docs/reference/field-types/color.md]: expanded options and pickerOptions sub props

**Review by abea — APPROVED** [review]

**Pull request merged**